web-common/src/features/alerts/criteria-tab/parseCriteriaError.ts

```diff
@@ -0,0 +1,10 @@
+// criteria[0].value must be a `number` type, but the…inal value was: `NaN` (cast from the value `""`).
```

Comment:
Typo: "the...inal"

Reply:
This was just for my reference. Will remove this comment.